### PR TITLE
Containerapp 0.3.11 Release 

### PR DESCRIPTION
**Pull request by StrawnSC**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Containerapp

File: src/containerapp/azext_containerapp/_clients.py [hunk 1]
```diff
@@ -15,6 +15,8 @@
 logger = get_logger(__name__)
 
 STABLE_API_VERSION = "2022-03-01"
```
Comment (panchagnula) [inline src/containerapp/azext_containerapp/_clients.py:17]: Nit: if stable api-version isn't being used any where can we remove this constant?

**Review by panchagnula — APPROVED** [review]
one comment - otherwise looks good.

**Comment by panchagnula** [comment]
@zhoxing-ms can we get your initial review on this, so we can release this soon. Thanks!

File: src/containerapp/azext_containerapp/_params.py [hunk 2]
```diff
@@ -77,13 +77,22 @@ def load_arguments(self, _):
...
         c.argument('dapr_app_port', type=int, help="The port Dapr uses to talk to the application.")
         c.argument('dapr_app_id', help="The Dapr application identifier.")
         c.argument('dapr_app_protocol', arg_type=get_enum_type(['http', 'grpc']), help="The protocol Dapr uses to talk to the application.")
+        c.argument('dapr_http_read_buffer_size', options_list=['--dapr-http-read-buffer-size', '--dhrbs'], type=int, help="Dapr max size of http header read buffer in KB to handle when sending multi-KB headers..")
+        c.argument('dapr_http_max_request_size', options_list=['--dapr-http-max-request-size', '--dhmrs'], type=int, help="Increasing max size of request body http and grpc servers parameter in MB to handle uploading of big files.")
+        c.argument('dapr_log_level', arg_type=get_enum_type(["info", "debug", "warn", "error"]), help="Sets the log level for the Dapr sidecar.")
+        c.argument('dapr_enable_api_logging', options_list=['--dapr-enable-api-logging', '--dal'], help="Enables API logging for the Dapr sidecar.")
```
Comment (zhoxing-ms) [inline src/containerapp/azext_containerapp/_params.py:95]: Please use the first-person voice in the help message
```suggestion
        c.argument('dapr_http_max_request_size', options_list=['--dapr-http-max-request-size', '--dhmrs'], type=int, help="Increase max size of request body http and grpc servers parameter in MB to handle uploading of big files.")
        c.argument('dapr_log_level', arg_type=get_enum_type(["info", "debug", "warn", "error"]), help="Set the log level for the Dapr sidecar.")
        c.argument('dapr_enable_api_logging', options_list=['--dapr-enable-api-logging', '--dal'], help="Enable API logging for the Dapr sidecar.")
```

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**